portfolio/src/main/webapp/suggestions_script.js

```diff
@@ -11,37 +11,45 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
+ 
 document.addEventListener("DOMContentLoaded", async function() {
     let response = await fetch('/list-comments');
     let comments = await response.json();
+    console.log(comments);
     const commentArea = document.getElementById('comment-list');
     comments.forEach((comment) => {
+        console.log(comment);
       commentArea.appendChild(createComment(comment));
     });
   });
-
+ 
 function createComment(comment) {
-  const commentElement = document.createElement('li');
-  commentElement.className = 'comment';
-
-  const commentTextElement = document.createElement('span');
+  const commentElement = document.createElement("div");
+  commentElement.setAttribute("id", "comment");
```

Comment:
This gives each created comment an id of comment. perhaps you want to set this as a class instead?

commentElement.classList.add('comment');